index.rst

```diff
@@ -48,7 +48,353 @@
+data acquisition and release here.
+These dates are more authoritatively addressed in RTN-011.
+
+AuxTel/LATISS
+-------------
+
+Initially the data available for use with "PP Preview" will be AuxTel/LATISS data taken on the Summit.
+
+While it would be possible to do an AP-like processing of a simulated Rubin dataset, or a collection of imaging data from an ``afw``-compatible project such as DECam, the main point of the "PP Preview" is to exercise and demonstrate the ability to serve *progressively* the results of ongoing processing, so a one-time static processing of an existing data collection would not meet this need.
+
+An observing run with AuxTel is currently done approximately every two weeks.
+While over half of the time is devoted to spectroscopy, it appears that imaging data are still being taken on a regular basis.
+Imaging and spectroscopic data may be separated readily on the filter ID, which has a distinctive "grism" value for the latter.
+
+AuxTel imaging data is the primary initial target of "PP Preview".
+
+ComCam
+------
+
+Since it now appears likely that the initial commissioning of the main telescope will use ComCam after all,
+this will become a key target for "PP Preview" processing,
+with a goal of having a suitable data flow from data acquisition through RSP access in place as early as
+possible in the ComCam data acquisition era.
+
+Delivery of AuxTel imaging data through "PP Preview" will continue into the ComCam era and will provide valuable
+practice in working with multiple datasets.
+
+While existing RSP tooling is usable for the display of a ComCam array's worth of images -- 9 CCDs -- at once,
+that will not be the case for LSSTCam.
+Therefore, one aim of the ComCam era of "PP Preview" should be to exercise tooling for visualization of
+focal-plane arrays of data in a way that *will* scale to LSSTCam, even though it may not be strictly
+needed on this time scale.
+This is briefly discussed further below, but the planned work in this area is yet to be defined in detail.
+
+LSSTCam
+-------
+
+Once commissioning switches to LSSTCam, we anticipate continuing "PP Preview" service as soon as possible.
+This will depend on successfully scaling up all the processing and data service elements to the new load.
+Commissioning a user-facing full-focal-plane display capability will be a key goal of this period in
+addition to all of the pure scaling issues that will be involved.
+
+As noted elsewhere, "PP Preview" service of LSSTCam to staff is in addition to and in parallel with
+any Data Preview 1 service of a limited set of commissioning data to users.
+
+
+Data Processing and Data Products
+=================================
+
+The aim is to always be in the position to perform some level of automated, AP-like, processing of all imaging data taken, regardless of whether templates are available or not for the section of sky being observed.
```

Comment:
Perhaps rephrase to "perform automated execution of some subset of the AP processing for all imaging data taken"